ui/src/app/views/cohort-list/component.html

```diff
@@ -53,7 +53,11 @@ <h3 class="cohorts-header">Cohorts</h3>
             </button>
           </clr-dropdown-menu>
         </clr-dropdown>
-        <div class="name">{{cohort.name}}</div>
+        <div class="name"
+             [class.disabled]="actionsDisabled"
+             (click)="actionsDisabled || openCohort(cohort)">
```

Comment:
Probably don't need the `actionsDisabled` here since you also check in the callback.